CHANGELOG.md

```diff
@@ -1,5 +1,78 @@
 # Change Log
 
+## [0.22.0](https://github.com/redhat-developer/vscode-xml/milestone/30?closed=1) (October 20, 2022)
+
+### Enhancements
+
+ * Support for Relax NG. See [lemminx#237](https://github.com/eclipse/lemminx/issues/237).
+ * RelaxNG compact textmate+language-configuration support. See [#781](https://github.com/redhat-developer/vscode-xml/pull/781).
+ * Support `xml.format.closingBracketNewLine` setting with experimental formatter. See [lemminx#1247](https://github.com/eclipse/lemminx/issues/1247).
+ * Support `xml.format.xsiSchemaLocationSplit` setting with experimental formatter. See [lemminx#1246](https://github.com/eclipse/lemminx/issues/1246).
+ * Support `xml.format.spaceBeforeEmptyCloseTag` setting with experimental formatter. See [lemminx#1245](https://github.com/eclipse/lemminx/issues/1245).
+ * Support `xml.format.joinContentLines` setting with experimental formatter. See [lemminx#1244](https://github.com/eclipse/lemminx/issues/1244).
+ * Support `xml.format.joinCommentLines` setting with experimental formatter. See [lemminx#1243](https://github.com/eclipse/lemminx/issues/1243).
+ * Support `xml.format.preserveEmptyContent` setting with experimental formatter. See [lemminx#1242](https://github.com/eclipse/lemminx/issues/1242).
+ * Support `xml.format.joinCDATALines` setting with experimental formatter. See [lemminx#1241](https://github.com/eclipse/lemminx/issues/1241).
+ * Support `xml.format.preservedNewlines` setting with experimental formatter. See [lemminx#1240](https://github.com/eclipse/lemminx/issues/1240).
+ * Support `xml.format.enforceQuoteStyle` setting with experimental formatter. See [lemminx#1239](https://github.com/eclipse/lemminx/issues/1239).
+ * Support `xml.format.emptyElements` setting with experimental formatter. See [lemminx#1238](https://github.com/eclipse/lemminx/issues/1238).
+ * Set `xml.format.xsiSchemaLocationSplit` as onPair by default. See [#749](https://github.com/redhat-developer/vscode-xml/issues/749).
+ * Provide comment formatting for experimental formatter. See [lemminx#1259](https://github.com/eclipse/lemminx/issues/1259).
+ * Update description for joinCommentLines setting with experimental formatter. See [#746](https://github.com/redhat-developer/vscode-xml/pull/746).
+ * Update description for enforceQuoteStyle setting with experimental formatter. See [#737](https://github.com/redhat-developer/vscode-xml/pull/737).
```

Comment:
No need to mention these. I don't think updating a description is as high value for users.